File: week4/Constructor.java
import java.util.Scanner;
class Student
{
  public
    Student()
    {
      String collegeName="MVGR";
      int collegeCode=33;
      System.out.print("CollegeName : "+collegeName+"\n"+"CollegeCode : "+collegeCode+"\n");
    }
    Student(String fullName,double semPercentage)
    {
      System.out.print("FullName : "+fullName+"\n"+"SemPercentage : "+semPercentage);
    }
}
public class Constructor
{
  public static void main(String[] args)
  {
    Student SriVyshnavi=new Student();
    Scanner input=new Scanner(System.in);
    System.out.print("FullName : ");
    String fullName=input.nextLine();
    System.out.print("SemPercentage : ");
    double semPercentage=input.nextDouble();
    input.close();
    Student Vyshnavi=new Student(fullName,semPercentage);
  }
}

File: week3/constructor.cpp
#include<iostream>
using namespace std;

class Student
{
    public:
        Student()
        {
            string collegeName = "MVGR";
            int collegeCode=050;
            cout<<"CollegeName : "<<collegeName<<endl;
            cout<<"CollegeCode: "<<collegeCode<<endl;
        }
        Student(string fullName, double semPerentage)
        {
            cout<<"FullName : "<<fullName<<endl;
            cout<<"SemPercentage : "<<semPerentage<<endl;
        }
        ~Student(){
         cout<<"I'M GONE!!!!"<<endl; 
        }
};
int main()
{
    Student SRI VYSHNAVI;
    string name;
    float val;
    cout<<"Enter your name : ";
    getline(cin,name);
    cout<<"Enter your sem percentage : ";
    cin>>val;
    Student SriVyshnai05(name,val);
    return 0;
}
